docs/configuration.md

```diff
@@ -200,10 +200,10 @@ Default: `['clear-text', 'progress', 'html']`<br />
 Command line: `--reporters clear-text,progress,dots,dashboard,html,json`<br />
 Config file: `"reporters": ["clear-text", "progress", "dots", "dashboard", "html", "json"]`
 
-With `reporters`, you can set the reporters for stryker to use.
+With `reporters`, you can set the reporters for Stryker to use.
```

Comment:
Consistent capitalisation of the product name.

Reply:
Yeah, true! Thanks!